packages/slice-machine/lib/models/ui/Slice/index.tsx

```diff
@@ -201,11 +239,15 @@ export const SharedSlice = {
           // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-argument
           sx={bordered ? borderedSx(sx) : defaultSx(sx)}
         >
-          <SliceThumbnail
-            withShadow={false}
-            screenshotUrl={screenshotUrl}
-            heightInPx={thumbnailHeightPx}
-          />
+          {wrapperType === WrapperType.changesPage ? (
```

Comment:
if we have the same condition on the whole content of the card, maybe we could do something about it.

Reply:
ah yes didn't realise there were 2 consecutive `WrapperType.changesPage` checks. I have joined them for clarity. Think it might be fine to keep it like this tho since a lot of component has logic needed in both cases